tst/EnergyPlus/unit/OutputReportTabular.unit.cc

```diff
@@ -11650,7 +11650,7 @@ TEST_F(SQLiteFixture, DOASDirectToZone_ZoneMultiplierRemoved)
                                        "  AND RowName = 'DOAS Direct to Zone'");
     // check the value from result records
     Real64 return_val_sensible_instant = execAndReturnFirstDouble(query_sensible_instant);
-    EXPECT_EQ(return_val_sensible_instant, 517.05);
+    EXPECT_EQ(return_val_sensible_instant, 600.28);
```

Comment:
Corrected the test values due to the introduction of missing standard air density in the DOAS load calculation.